README.md

```diff
@@ -33,6 +33,10 @@ Differences noted here
 |Capability|Description|Values|
 |----------|-----------|------|
 |`noReset`|Do not destroy or shut down sim after test. Start tests running on whichever sim is running, or device is plugged in. Default `false`|`true`, `false`|
+|`processArguments`|This should be passed as object like this let proArgs = {
```

Comment:
This ends up being formatted weirdly.

https://github.com/yaphatak/appium-xcuitest-driver/blob/4ef2f7026a12dd80b90076b358326a14448451ff/README.md#usage